net-libs/gloox/gloox-1.0.28.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+MY_P="${P/_/-}"
+DESCRIPTION="A portable high-level Jabber/XMPP client library for C++"
+HOMEPAGE="https://camaya.net/gloox/"
+SRC_URI="https://camaya.net/download/${MY_P}.tar.bz2"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="GPL-3"
+# Check upstream changelog: https://camaya.net/gloox/changelog/
+# REVIEW(fkobi): I see no mention of 18 there and no change
+# from 1.0.24 to 1.0.28 in https://camaya.net/api/gloox-1.0.2[48]/
```

Comment:
You can add your notes as comments in Github. No need to add these to ebuilds where they have to be removed.

That being said, the library *libgloox.so.18* did have new functions added, but no SONAME change and no breaking changes to the library, so leaving the subslot at 18 should be fine here. 

You can see them using **abidiff** from the dev-util/libabigail package, or by using **iwdevtools**:
```
# abidiff /var/tmp/portage/net-libs/gloox-1.0.24-r1/image/usr/lib64/libgloox.so.18 /var/tmp/portage/net-libs/gloox-1.0.28/image/usr/lib64/libgloox.so.18
Functions changes summary: 0 Removed, 0 Changed, 0 Added function
Variables changes summary: 0 Removed, 0 Changed, 0 Added variable
Function symbols changes summary: 0 Removed, 5 Added function symbols not referenced by debug info
Variable symbols changes summary: 0 Removed, 0 Added variable symbol not referenced by debug info

5 Added function symbols not referenced by debug info:

  [A] _ZN5gloox17MUCMessageSession4sendERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEES8_RKNS1_4listIPKNS_15StanzaExtensionESaISC_EEE
  [A] _ZN5gloox7MUCRoom4sendERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEES8_RKNS1_4listIPKNS_15StanzaExtensionESaISC_EEE
  [A] _ZNK5gloox10TLSDefault18channelBindingTypeB5cxx11Ev
  [A] _ZNK5gloox13OpenSSLClient18channelBindingTypeB5cxx11Ev
  [A] _ZNK5gloox7TLSBase18channelBindingTypeB5cxx11Ev

```